temba/channels/models.py

```diff
@@ -1833,7 +1833,7 @@ def send_vumi_message(cls, channel, msg, text):
 
         api_url_base = channel.config.get('api_url', cls.VUMI_GO_API_URL)
 
-        url = urlparse.urljoin(api_url_base, "%s/messages.json" % channel.config['conversation_key'])
+        url = "%s/%s/messages.json" % (api_url_base, channel.config['conversation_key'])
```

Comment:
Can you add a test if there isn't one confirming this url is consistent? It was actually a change to this line that broke vumi sms in production for us from the last PR.